From 6f9c64410703baa0d6aed1be92f9b3d805c9922d Mon Sep 17 00:00:00 2001
From: Thomas Langer <thomas.langer@intel.com>
Date: Fri, 18 Jan 2019 18:13:28 +0100
Subject: [PATCH] Silent warning if phy has no irq

It is valid to use external phys without interrupts connected.
---
 drivers/net/ethernet/lantiq/ltq_eth_drv_xrx500.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/lantiq/ltq_eth_drv_xrx500.c b/drivers/net/ethernet/lantiq/ltq_eth_drv_xrx500.c
index 390ebffdf46c..8f406d83d089 100644
--- a/drivers/net/ethernet/lantiq/ltq_eth_drv_xrx500.c
+++ b/drivers/net/ethernet/lantiq/ltq_eth_drv_xrx500.c
@@ -1527,7 +1527,7 @@ xrx500_of_port(struct net_device *dev, struct device_node *port)
 		if (of_irq_to_resource_table(port, &irqres, 1) == 1)
 			p->irq_num = irqres.start;
 		else
-			pr_info("couldn't get irq number for gphy !!\n");
+			pr_debug("couldn't get irq number for gphy !!\n");
 	}
 
 	p->num = *id;
